doc/extensions/tutorial.rst

```diff
@@ -157,6 +160,13 @@ the ``entry_points`` section like this::
         example_iauthfunctions=ckanext.iauthfunctions.plugin:ExampleIAuthFunctionsPlugin
     ''',
 
+.. note::
```

Comment:
Can you move this note to the very beginning of the section (under "Adding the plugin to setup.py") and change the text to:

> You can skip this section if you run the `ckan generate extension` command